### PR TITLE
#160094211 Get all created tags

### PR DESCRIPTION
**Pull request by Lumexralph**
#### What does this PR do?
Get all tags created
#### Description of Task to be completed?
    - Users should be able to get the list of all tags associated with articles
    - add unit tests
#### How should this be manually tested?
In postman:
To get all tags: GET `http://localhost:3000/api/tags`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160094211

### CODE REVIEW

File: tests/controllers/articleController.test.js [hunk 1]
```diff
@@ -384,4 +396,16 @@ describe('Articles API endpoints', () => {
         done();
       });
   });
+
+  it('Should not get all tags created', (done) => {
```
Comment (OKiMaureen) [inline tests/controllers/articleController.test.js]: @Lumexralph, I think a better description would be 'should return 'no tags created' when tag size is  0.  The above description sounds like should not get all tags even after being created.

Comment (Lumexralph) [reply tests/controllers/articleController.test.js]: @OKiMaureen Thanks MO, what do you feel about using `should return an empty array when no tags are created` ?

Comment (OKiMaureen) [reply tests/controllers/articleController.test.js]: @Lumexralph yea..cool too.

**Review by veeqtor — APPROVED** [review]

**Review by OKiMaureen — APPROVED** [review]

**Review by anuonifade — APPROVED** [review]
LGTM

**Pull request merged**